opencompass/multimodal/models/minigpt_4/prompt_constructor.py

```diff
@@ -53,3 +53,45 @@ def _process(self, data_samples: List[DataSample]) -> str:
         else:
             prompt = self.image_prompt + ' ' + question + ' ' + option + ' ' + self.reply_prompt  # noqa
         return prompt
+
+
+class MiniGPT4MMEPromptConstructor:
+    """Prompt constructor for MiniGPT-4 on MME.
+
+    Args:
+        image_prompt (str): Image prompt.
```

Comment:
```suggestion
        image_prompt (str): Image prompt. Defaults to `''`.
```
Please also check other parts and make changes accordingly.

Reply:
Done. Add default value to both mmbench and mme prompt constructors.